api/helpers/gamestate/publish-game-state.js

```diff
@@ -44,7 +44,14 @@ module.exports = {
 
       //Last Event, and Extra Socket Variables
       const happened = gameState.moveType !== MoveType.FIZZLE;
-      const { playedBy } = gameState;
+      let { playedBy } = gameState;
+
+      // When resolving, the player who resolves is the opponent
+      // of the one who played the one-off (if the one-off doesn't fizzle)
+      if ( gameState.moveType === MoveType.RESOLVE ) {
+        playedBy = (playedBy + 1) % 2;
+      }
```

Comment:
This feels hacky and I'm not sure what's best. Currently the legacy `/resolve` endpoint emits a socket event with a `playedBy` value that's the _opposite_ of whoever called `resolve`. The idea is that if a one-off does resolve without being countered, then the person who calls resolve is the opponent of the person who played the one-off.

This let's the clientside somewhat cleanly interpret who played the oneOff as `playedBy`, but it's somewhat inconsistent with the way playedBy is used in the rest of the GameState API. Alternatively we could try to rework how it's represented clientside in `inGameEvents`. I don't love either option at the moment

Reply:
Notably this is what the client currently expects and overall we are making minimal changes to the client code for the MVP rollout of the GameState API in the name of downscoping. So when we rework the client data structures this will change regardless, but for now it might be easiest to keep this consistent with legacy?

Reply:
I think this makes sense for now, but maybe it's better to do it inside the `resolve` helper?